### ISSUE TITLE
Project technical setup

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#21]
## User story
1. As a {software developer}
2. I want / need {the basic technical systems set up}
3. So that {I can efficiently and effectively develop the frontend and backend of the DIdentity software}

## Acceptance criteria
* the programming environments needed for the project are set up
* data storage set up
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
